server/README.md

```
@@ -20,6 +20,93 @@ Step 4: To run the Kommunicate-Web-SDK in local you need to run this command in
+
+#### Required Dependencies
+Ensure the following dependencies are installed:
+```sh
+npm install --save-dev prettier husky lint-staged
+```
+
+
+
+## Troubleshooting
+
+#### **1. No Valid Configuration Found**
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)